examples/feedback_review/Ballerina.toml

```
@@ -0,0 +1,14 @@
+[package]
+org = "irash"
```

Let's change this to WSO2.

```suggestion
org = "wso2"
```